app/services/cfe/assets_payload_service.rb

```diff
@@ -38,10 +38,10 @@ def properties
         property_entry_form = instantiate_form(PropertyEntryForm)
         main_home = {
           value: property_entry_form.house_value,
-          outstanding_mortgage: (property_entry_form.mortgage if property_entry_form.owned_with_mortgage?) || 0,
+          outstanding_mortgage: (property_entry_form.mortgage if property_entry_form.owned_with_mortgage? || property_entry_form.shared_ownership?) || 0,
```

Comment:
i think this additional OR statement is required here but would be good if someone confirms my thinking